spec/handlers/newflow/student_signup_spec.rb

```diff
@@ -69,14 +69,76 @@ module Newflow
             hash_including({ email_address: an_instance_of(EmailAddress) })
           )
         )
-        result
+        subject
       end
 
       it 'stores selection in User whether to receive newsletter or not' do
         expect(User.new.receive_newsletter).to be_falsey
-        result
+        subject
         expect(User.last.receive_newsletter).to be(true)
       end
+
+      it 'outputs a user' do
+        expect(subject.outputs.user).to be_present
+      end
+    end
+
+    context 'success -- when passed-in a user_from_signed_params' do
+      # a user that already had an account, then tries to sign up using signed params (so via an LMS),
```

Comment:
imo, context should be all u need here for comments...it should say what you're saying in the comment.  comments are rarely needed ever in code, code (esp ruby dsls like rspec, should read like prose.

Reply:
True. The comment is/was more descriptive than the context. Moved the comment into the context.

Reply:
Very true. Fixed.